src/status_im/contexts/wallet/signals.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.contexts.wallet.signals
```

Comment:
should I just put this in a regular effects file? not sure if we have a common approach for this but seemed nice to me to somewhat separate it out 👍

Reply:
A separate ns is good as the signals for the wallet will grow over time. 👍